person.rb

```diff
@@ -1,14 +1,21 @@
+require './nameable'
 attr_accessor :name, :age
+
 attr_reader :id
```

Comment:
- [ ] These helper methods are supposed to be used within the class. Please consider moving them! 🤓 🔧